packages/components/src/internal/components/domainproperties/Lookup/Fields.tsx

```diff
@@ -203,7 +204,7 @@ class TargetTableSelectImpl extends React.Component<TargetTableSelectProps, ITar
     }
 
     render() {
-        const { id, onChange, value, name, disabled } = this.props;
+        const { id, onChange, value, name, disabled, shouldDisableNonExists = true } = this.props;
```

Comment:
nit: For class-based components I'd recommend using `defaultProps` to configure this so that other methods on this class don't accidentally receive the wrong default value.

Reply:
ah yes, oops. thanks.